README.md

```diff
@@ -52,136 +52,68 @@ npm run start
 
 ## RESTful CRUD API
 
-### POST: Creates a new review
+### POST: Creates a new review for a property
 
-URL /api/listings/:review_id
+URL /api/:property/review
 
-Example: /api/listings/1
+Example: /api/10000/review
```

Comment:
I kept the listings in both my APIs so /api/listings/somethingelse.

I don't this will cause any issues but may be nice to be consistent across the team. We can discuss this live.